### PR TITLE
CLM-28680 - Require Java 11

### PR DESCRIPTION
**Pull request by koraytugay**
https://sonatype.atlassian.net/browse/CLM-28680
https://jenkins.ci.sonatype.dev/job/integrations/job/jenkins/job/feature-snapshots/job/CLM-28680_Require_Java_11/

### CODE REVIEW

File: Jenkinsfile.sonatype.extra-tests [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 String agentLabel = 'ubuntu-zion'
 
-def runTests(String javaVersion = 'Java 8', String mavenExtra = '') {
+def runTests(String javaVersion = 'OpenJDK 11', String mavenExtra = '') {
```
Comment (vladt) [inline Jenkinsfile.sonatype.extra-tests:10]: Do we need two executions for Java 11?

Comment (koraytugay) [reply Jenkinsfile.sonatype.extra-tests:10]: Hey Vlad, I do not understand this comment. This is only the default value if `runTests` is not called with a specific version?

Comment (vladt) [reply Jenkinsfile.sonatype.extra-tests:10]: > Hey Vlad, I do not understand this comment. This is only the default value if runTests is not called with a specific version?

Sorry, I commented on the wrong line. It should have been on line 36. There are two stages for Java 11.

Comment (koraytugay) [reply Jenkinsfile.sonatype.extra-tests:10]: @vladt Made minimum supported version `2.361.1`.

**Review by eduard-tita — APPROVED** [review]
+1

File: pom.xml [hunk 2]
```diff
@@ -73,11 +73,11 @@
     <gitHubRepo>jenkinsci/nexus-platform-plugin</gitHubRepo>
 
     <!-- This is the minimum advertised compatible version -->
-    <jenkins.version>2.346.3</jenkins.version>
+    <jenkins.version>2.361.1</jenkins.version>
 
     <!-- See: https://github.com/jenkinsci/bom -->
-    <jenkins.tools.bom.artifactId>bom-2.346.x</jenkins.tools.bom.artifactId>
-    <jenkins.tools.bom.version>1580.v47b_429a_c853a</jenkins.tools.bom.version>
+    <jenkins.tools.bom.artifactId>bom-2.361.x</jenkins.tools.bom.artifactId>
```
Comment (vladt) [inline pom.xml:79]: I thin we need to also set:
    <maven.compiler.source>11</maven.compiler.source>
    <maven.compiler.target>11</maven.compiler.target>

Comment (koraytugay) [reply pom.xml:82]: 52b23badee845196e0d10200dfc847525dfe89ab
